android/src/main/java/expo/modules/textextractor/ExpoTextExtractorModule.kt

```diff
@@ -33,7 +46,7 @@ class ExpoTextExtractorModule : Module() {
             promise.reject(CodedException("err", error))
           }
       } catch (error: Exception) {
-        promise.reject(CodedException("err", error))
+        promise.reject(CodedException("err", error.message ?: "Unknown error", error))
```

Comment:
Maybe make the error code more usefull.
```suggestion
        promise.reject(CodedException("UNKNOWN_ERROR", error.message ?: "Unknown error", error))
```